src/index.ts

```diff
@@ -200,7 +200,18 @@ export class ProjectConfig {
     const validator = new jsonschema.Validator();
     const result = validator.validate(configJsonObject, getSchema());
     if (result.errors.length > 0) {
-      throw result.errors[0];
+      const error = result.errors[0]!;
+      let message;
+      if (error.property && error.message) {
+        let propertyName = error.property;
+        if (propertyName.startsWith('instance.')) {
+          propertyName = propertyName.slice(9);
+        }
+        message = `Property '${propertyName}' ${error.message}`;
+      } else {
+        message = error.toString()
```

Comment:
in this case, just throw the original error so that we don't lose the stack trace

Reply:
+1, done

Reply:
yea, very helpful for catching, handling, then rethrowing if needed